docs/self-managed/backup-restore/backup-and-restore.md

```diff
@@ -39,20 +43,17 @@ PUT /_snapshot/my_repository/camunda_zeebe_records_backup_x
 }
 ```
 
-By default, the indices are prefixed with `zeebe-record`. If you have configured a different prefix when configuring elasticsearch exporter in Zeebe, use that instead.
-
-9. Take Backup `x` of Zeebe. (See [How to take Zeebe backup](self-managed/backup-restore/zeebe-backup-and-restore.md))
-10. Wait until Backup `x` of exported zeebe records is complete.
-11. Wait until Backup `x` of Zeebe is completed. (See [How to monitor Zeebe backup](self-managed/backup-restore/zeebe-backup-and-restore.md))
-12. Resume exporting in Zeebe. (See [Zeebe management api](/self-managed/zeebe-deployment/operations/management-api.md))
+By default, the indices are prefixed with `zeebe-record`. If you have configured a different prefix when configuring Elasticsearch exporter in Zeebe, use this instead. 9. Take a backup `x` of Zeebe. See [how to take a Zeebe backup](self-managed/backup-restore/zeebe-backup-and-restore.md). Wait until the backup `x` of the exported Zeebe records is complete and wait until the backup `x` of Zeebe is completed before proceeding. See [how to monitor a Zeebe backup](self-managed/backup-restore/zeebe-backup-and-restore.md). 10. Resume exporting in Zeebe. See [Zeebe management API](/self-managed/zeebe-deployment/operations/management-api.md).
```

Comment:
Steps 9-12 should be in bullet points.

Reply:
Nice catch on the typo 👍 Adjusted in the latest commit.